git.mk

```diff
@@ -3,13 +3,15 @@ $(error Use the main Makefile)
 endif
 
 SUBPROJECTS += git
-GIT_VERSION := 2.31.1
-DEB_GIT_V   ?= $(GIT_VERSION)-1
+GIT_VERSION := 2.32.0
+DEB_GIT_V   ?= $(GIT_VERSION)
 
 GIT_ARGS += uname_S=Darwin \
 	HOST_CPU=$(GNU_HOST_TRIPLE) \
 	DESTDIR=$(BUILD_STAGE)/git \
 	MANDIR=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man \
+	PYTHON_PATH=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/python3
+	ASCIIDOC_PATH=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/asciidoc
```

Comment:
For the record, you need to add an \ after this arg.

Reply:
Surprising it worked fine without the \, but I will add.